scripts/datajoint_to_nwb.py

```diff
@@ -0,0 +1,190 @@
+        description=subj['subject_description'],
+        genotype=' x '.join((subject.Subject.Allele & session_key).fetch('allele')),
+        sex=subj['sex'],
+        species=subj['species'])
+
+    # =============== Extracellular ====================
+    probe_insertion = ((extracellular.ProbeInsertion & session_key).fetch1()
+                       if extracellular.ProbeInsertion & session_key
+                       else None)
+    if probe_insertion:
+        probe = nwbfile.create_device(name = probe_insertion['probe_name'])
+        electrode_group = nwbfile.create_electrode_group(
+            name='; '.join([f'{probe_insertion["probe_name"]}: {str(probe_insertion["channel_counts"])}']),
+            description = 'N/A',
+            device = probe,
+            location = '; '.join([f'{k}: {str(v)}' for k, v in
+                                  (reference.BrainLocation & probe_insertion).fetch1().items()]))
+
+        for chn in (reference.Probe.Channel & probe_insertion).fetch(as_dict=True):
+            nwbfile.add_electrode(id=chn['channel_id'],
+                                  group=electrode_group,
+                                  filtering=hardware_filter,
+                                  imp=-1.,
+                                  x=0.0,  # not available from data
+                                  y=0.0,  # not available from data
+                                  z=0.0,  # not available from data
+                                  location=electrode_group.location)
+
+        # --- unit spike times ---
+        nwbfile.add_unit_column(name='depth', description='depth this unit')
+        nwbfile.add_unit_column(name='quality', description='spike width of this unit')
+        nwbfile.add_unit_column(name='cell_type', description='cell type (e.g. wide width, narrow width spiking)')
+
+        for unit in (extracellular.UnitSpikeTimes & probe_insertion).fetch(as_dict=True):
+            # make an electrode table region (which electrode(s) is this unit coming from)
+            nwbfile.add_unit(id=unit['unit_id'],
+                             electrodes=(unit['channel_id']
+                                         if isinstance(unit['channel_id'], np.ndarray) else [unit['channel_id']]),
+                             depth=unit['unit_depth'],
+                             quality=unit['unit_quality'],
+                             cell_type=unit['unit_cell_type'],
+                             spike_times=unit['spike_times'])
+
+    # =============== Behavior ====================
+    behavior_data = ((behavior.LickTimes & session_key).fetch1()
+                     if behavior.LickTimes & session_key
+                     else None)
+    if behavior_data:
+        behav_acq = pynwb.behavior.BehavioralEvents(name = 'lick_times')
+        nwbfile.add_acquisition(behav_acq)
+        [behavior_data.pop(k) for k in behavior.LickTimes.primary_key]
+        for b_k, b_v in behavior_data.items():
```

Comment:
Use descriptive variable names
```suggestion
        for timeseries_name, timestamps in behavior_data.items():
```